docs/src/basics/nomenclature.md

```diff
@@ -0,0 +1,53 @@
+<h1>Nomenclature</h1>
+
+This section defines commonly used terminology referenced within the Zeebe documentation.
+
+**Workflow**:  A workflow is a defined sequence of distinct steps representing your business logic.  Examples of a workflow could be making e-commerce shopping experience, a process to onboard a new employee, etc.  In Zeebe nomenclature, workflows are identified by a unique name and bpmn process id.  The workflow is usually also referred to as the BPMN model.
+
+_Workflow Instance_:  While a workflow represents a defined sequence of distinct steps representing your business logic, a workflow instance represents a currently executing or completing workflow.  For a single workflow, there could be many associated workflow instances in various stages of their executing cycle.   Workflow instances are identitied by workflow instance id.  Executing workflows are also sometimes referred to as inflight workflows.
+
+_Workflow Variable_:  A workflow variable represents the execution state of a workflow instance.  The variables capture business process parameters which are input and output of various stages of the workflow instance and which also influence process flow execution.
+
+_Workflow Deployment_: a workflow definition cannot execute unless the broker is aware of its existence. This is done by deploying a committing a workflow definition to the broker. The act of pushing of broker is referred to as workflow deployment.
+
+_Broker_:  A broker is an instance of a Zeebe installation which executes workflows and manages workflow state.  A single broker will be installed on a single machine.
+
+_Cluster_:  A cluster represents a configuration of one or more brokers collaborating to execute workflows.  Each broker in a cluster acts as a leader or a follower.  
+
+_Leader_:  In a clustered environment, one broker tis responsible for workflow execution and housekeeping.  Housekeeping includes, taking snapshots, replication and running exports for a given partition.
+
+_Follower_:  In a clustered environment, a broker which is not a leader is a follower of a given partition.  A follower assumes leader responsibilities when the leader goes down.
+
+_Record_: A Zeebe record is a collection of workflow variables and other state transitions of an executing workflow instance at a given point in time.  During execution of a workflow instance numerous records will be generated to capture the various state transitions from start to workflow instance completion.
+
+_Event_:  Within the context of Zeebe, an event represent a state transition associated with an executing workflow instance.  Events include variable changes, changes to transition elements in the workflow model, etc.   An event will be represented by a timestamp, the variable name and variable value.  These events are stored in an append-only log.
```

Comment:
We should also describe commands and that events are also reactions to a command.

Reply:
Please review the event and command section.